rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -172,6 +182,11 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    Date-Time:
+      description: the date-time notation as defined by RFC 3339, section 5.6
+      type: string
+      format: date-time
```

Comment:
❗️This format affects the parameter in the generated IngestAPI interface:
```java
@ApiParam(value = "The the last modified time of the product to be ingested. ", required = true) @RequestHeader(value = "Last-Modified", required = true) OffsetDateTime lastModified
```

Reply:
I don't think I understand what you mean... the format is mentioned here: https://swagger.io/docs/specification/data-models/data-types/#string

Reply:
The class for that parameter in IngestApi should be OffsetDateTime, not String.

Reply:
Ok, the issue is https://github.com/connexta/ion-ingest-api/pull/81/files#diff-82b258e5dd3434d2913f6aa7165c271bR83 
then :+1: will update